cwl/src/main/scala/cwl/CwlDecoder.scala

```diff
@@ -21,7 +21,7 @@ object CwlDecoder {
     val cwlToolResult =
       Try(CwltoolRunner.instance.salad(path))
         .toEither
-        .leftMap(t => NonEmptyList.one(s"running cwltool on file ${path.toString} failed with ${t.getMessage}"))
+        .leftMap(t => NonEmptyList.one(s"running cwltool on file ${path.toString} failed with ${t.value}"))
```

Comment:
What is `.value`? Could this line be simplified to `s"running cwltool on file $path failed with $t"`?

Reply:
Good call, not sure why I didn't try that